source/configure/authentication-configuration-settings.rst

```diff
@@ -411,7 +411,24 @@ AD/LDAP
 .. include:: ../_static/badges/ent-pro-cloud-selfhosted.rst
   :start-after: :nosearch:
 
-Access the following configuration settings in the System Console by going to **Authentication > AD/LDAP**.
+Access the following configuration settings in the System Console by going to **Authentication > AD/LDAP**. This opens the AD/LDAP setup wizard with step-by-step sections and testing to help configure each setting.
+
```

Comment:
Is the wizard always present (e.g., past the initial setup)?

Reply:
@streamer45 Yep, it's always there.  It's more of a "settings broken up into bite-sized chunks, which can be tested at each stage, but you can edit any part later and then retest it"... but Wizard is quicker.